### ISSUE TITLE
Broken copy-paste blockquotes scenario 

### ISSUE THREAD

**Issue by Violeta-Amza** [ProseMirror/prosemirror#1173]
### Issue details

The first blockquote node is not preserved when copying 2 blockquotes that contains a list as content. 

### Steps to reproduce

1. load https://prosemirror.net/
2. create 2 blockquotes with a simple list as content; see attached picture 
![input](https://user-images.githubusercontent.com/32900172/119300535-96a64880-bc69-11eb-87c2-a2680c93042d.JPG)
3. select the entire editor content using mouse, copy and paste it; as you can see in the attached picture, the first blockquote is not shown.
![output](https://user-images.githubusercontent.com/32900172/119300847-192f0800-bc6a-11eb-919e-c30fb04e4fcc.JPG)

Notes:
1. The html pasted contains both blockquotes.
```<html>
<body>
<!--StartFragment--><blockquote data-pm-slice="4 4 []"><ul><li><p>test</p></li></ul></blockquote><blockquote><ul><li><p>test</p></li></ul></blockquote><!--EndFragment-->
</body>
</html>
```
2. If blockquotes contain paragraph nodes than the pasted output is correct.
![scenario_2](https://user-images.githubusercontent.com/32900172/119302224-71670980-bc6c-11eb-9a8e-41ec034c73da.JPG)

### ProseMirror version

lastest

### Affected platforms

- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer
- [ ] Other

**Comment by marijnh** [comment]
What's happening is that, since your selection starts inside the first blockquote, the opening token for that blockquote isn't included in the clipboard content. This looks odd in this situation, but including it would also cause problems in other situations—if you select the end of a blockquote and paste it into another blockquote, you probably want both of them joined, rather than another level of quoting added, so there the opening token should not be included.

So basically, I think this is not really broken, but can understand that it looks unexpected. I'm not sure I want to further complicate the relevant code to work around that.

**Comment by Violeta-Amza** [comment]
I understand, thank you for your response.

**Comment by marijnh** [comment]
This issue has been moved to [issue #1173](https://code.haverbeke.berlin/prosemirror/prosemirror/issues/1173) on the [repository's new home](https://code.haverbeke.berlin/prosemirror/prosemirror/).
